{
	"about": "https://jvndb.jvn.jp/ja/contents/2023/JVNDB-2023-005725.html",
	"title": "wechat sdk python project の wechat sdk python における XML 外部エンティティの脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2023/JVNDB-2023-005725.html",
	"description": "wechat sdk python project の wechat sdk python には、XML 外部エンティティの脆弱性が存在します。",
	"identifier": "JVNDB-2023-005725",
	"references": [
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2018-25082",
			"source": "CVE",
			"id": "CVE-2018-25082"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2018-25082",
			"source": "NVD",
			"id": "CVE-2018-25082"
		},
		{
			"text": "https://github.com/zwczou/weixin-python/commit/e54abadc777715b6dcb545c13214d1dea63df6c9",
			"source": "関連文書",
			"id": "github.com (e54abadc777715b6dcb545c13214d1dea63df6c9)"
		},
		{
			"text": "https://github.com/zwczou/weixin-python/pull/30",
			"source": "関連文書",
			"id": "github.com (pull/30)"
		},
		{
			"text": "https://github.com/zwczou/weixin-python/releases/tag/v0.5.5",
			"source": "関連文書",
			"id": "github.com (v0.5.5)"
		},
		{
			"text": "https://vuldb.com/?id.223403",
			"source": "関連文書",
			"id": "vuldb.com (id.223403)"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/611.html",
			"id": "CWE-611",
			"title": "XML 外部エンティティ参照の不適切な制限(CWE-611)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:wechat_sdk_python_project:wechat_sdk_python",
			"version": "2.2",
			"vendor": "wechat sdk python project",
			"product": "wechat sdk python"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "6.5",
			"type": "Base",
			"severity": "Medium",
			"vector": "AV:N/AC:L/Au:S/C:P/I:P/A:P"
		},
		{
			"version": "3.0",
			"score": "9.8",
			"type": "Base",
			"severity": "Critical",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"date": "2023-11-09T16:16+09:00",
	"issued": "2023-11-09T16:16+09:00",
	"modified": "2023-11-09T16:16+09:00"
}
